python_bindings/src/halide/__init__.py

```diff
@@ -0,0 +1 @@
+from .halide_ import *
```

Comment:
apologies, but I don't recognize the lead-with-a-dot syntax, what does that indicate?